parsers/CA_YT.py

```diff
@@ -1,128 +1,119 @@
 #!/usr/bin/env python3
-
 from datetime import datetime
 from logging import Logger, getLogger
+from typing import Any
+from zoneinfo import ZoneInfo
 
-import arrow
 from bs4 import BeautifulSoup
 from requests import Session
 
-timezone = "America/Whitehorse"
+from electricitymap.contrib.lib.models.event_lists import ProductionBreakdownList
+from electricitymap.contrib.lib.models.events import ProductionMix, StorageMix
+from electricitymap.contrib.lib.types import ZoneKey
+from parsers.lib.exceptions import ParserException
+
+SOURCE = "www.yukonenergy.ca"
```

Comment:
While this contained www before I think it should just be the following:

```suggestion
SOURCE = "yukonenergy.ca"
```